lib/checks/report/report.rb

```diff
@@ -0,0 +1,28 @@
+module Checks
+  module Report
+    class Report
```

Comment:
Yep that's right.
I think special casing like that should be avoided as it is very fragile to future change (even more so than doing the dynamic loading in the first place)